### PR TITLE
Replaced 'save' run mode with 'no-email'

### LINKED ISSUE #114: Add run mode where HTML file is generated but no email is sent
Currently the `--save` flag saves an HTML file of the report being sent over email. However at present there is no way to save this file without sending an email. We want to extend and maybe even do a slight refactor of the Jeeves CLI to make this an option.

### PR DESCRIPTION
**Pull request by nathan-weinberg**
Fixes #114

### CODE REVIEW

**Comment by nathan-weinberg** [comment]
Please DNM even if approved - I need to make some compatibility changes with downstream `jeeves_config` repo to coincide with CLI changes

**Comment by nathan-weinberg** [comment]
@gouthampacha Yes I did some testing on my machine

File: report.py [hunk 1]
```diff
@@ -182,42 +182,44 @@ def run_report(config, blockers, server, header, test, save):
...
 	)
 
-	# parse list of email addresses
-	if test:
-		recipients = config['email_to_test'].split(',')
-	else:
-		recipients = config['email_to'].split(',')
+	# save HTML report to file
+	generate_html_file(htmlcode)
+	print('HTML file generated in "archives" folder')
```
Comment (EliadCohen) [inline report.py:187]: nit: folder name appears to be "archive"
```suggestion
	print('HTML file generated in "archive" folder')
```

File: report.py [hunk 2]
```diff
@@ -182,42 +182,44 @@ def run_report(config, blockers, server, header, test, save):
...
-		if save:
-			generate_html_file(htmlcode)
+				# log success if all recipients recieved report, otherwise raise exception
+				if response == {}:
+					print("Report successfully accepted by mail server for delivery")
+				else:
+					raise Exception("Mail server cannot deliver report to following recipients: {}".format(response))
+
+		except Exception as e:
+			print('Error sending email report: {}\nSee HTML file saved in "archives" folder'.format(e))
```
Comment (EliadCohen) [inline report.py:225]: Same as my prev nit

```suggestion
			print('Error sending email report: {}\nSee HTML file saved in "archive" folder'.format(e))
```

**Pull request merged**